arch/arm/soc/nordic_nrf/include/nrf_common.h

```diff
@@ -62,29 +62,29 @@
 #define NRF52_IRQ_FPU_IRQn                     38
 
 /**
- * @file UART baudrate divisors for NRF51/NRF52 family processors.
+ * @file UART baudrate divisors for nRF5x family processors.
  *
  * Based on Nordic MDK included header file: nrf52_bitfields.h
```

Comment:
This comment needs an improvement. If we are listing the divisors that are applicable for nRFx SOCs, i.e. nRF51 and nRF52 we need to add here the nrf51_bitfields.h, for consistency.

Reply:
The values in nrf51_bitfields.h were less accurate than the ones for nRF52, that's why we settled on those.